PINCache/PINOperationQueue.h

```diff
@@ -23,6 +23,16 @@ typedef NS_ENUM(NSUInteger, PINOperationQueuePriority) {
 - (instancetype)initWithMaxConcurrentOperations:(NSUInteger)maxConcurrentOperations concurrentQueue:(dispatch_queue_t)concurrentQueue NS_DESIGNATED_INITIALIZER;
 - (id <PINOperationReference>)addOperation:(dispatch_block_t)operation withPriority:(PINOperationQueuePriority)priority;
 - (void)cancelOperation:(id <PINOperationReference>)operationReference;
+
+/**
+ * Blocks the current thread until all of the receiver’s queued and executing operations finish executing.
+ *
+ * @discussion When called, this method blocks the current thread and waits for the receiver’s current and queued
+ * operations to finish executing. While the current thread is blocked, the receiver continues to launch already
+ * queued operations and monitor those that are executing.
```

Comment:
Despite `NSOperationQueue` has the same problem and does not have any of that comment I think it can be valuable to have one. So I added the comment